src/puppetlabs/websockets/client.clj

```diff
@@ -0,0 +1,10 @@
+(ns puppetlabs.websockets.client)
+
+(defprotocol WebSocketProtocol
```

Comment:
I think it would be nice to add some docstrings to this protocol and its functions to explain what they are/what they do, and also what this protocol is being used for in terms of TK Jetty9 implementation details.